conf/deployment/rosa/private_link_deployment.yaml

```diff
@@ -1,4 +1,3 @@
 ---
 ENV_DATA:
   private_link: True
-  machine-cidr: '10.1.0.0/16'
```

Comment:
This change needs a successful privatelink deployment job for verification. As far as I can see, the privatelink job was https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/20431/ but it failed

Reply:
This failed after deploying the provider cluster while accessing cluster. As I missed mentioning proxy
which result in ```2023-02-08 14:14:44  08:44:44 - MainThread - ocs_ci.utility.utils - WARNING  - Command stderr: Unable to connect to the server: dial tcp: lookup api.sgatfane-pl-pr.zn1a.s1.devshift.org on 10.11.5.160:53: no such host
2023-02-08 14:14:44  08:44:44 - MainThread - ocs_ci.ocs.openshift_ops - ERROR  - Cluster is not ready to use: Error during execution of the command: oc cluster-info.```
However, the cluster got deployed successfully,

This is today's job: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/20494/console